backend/src/main/java/com/woowacourse/gongcheck/application/SlackService.java

```diff
@@ -0,0 +1,25 @@
+package com.woowacourse.gongcheck.application;
+
+import com.slack.api.Slack;
+import com.slack.api.webhook.Payload;
+import com.woowacourse.gongcheck.application.response.Attachments;
+import com.woowacourse.gongcheck.application.response.SubmissionResponse;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SlackService implements AlertService {
+
+    @Override
+    @Async
```

Comment:
```suggestion
    @Async
    @Override
```
`@Override`는 어짜피 제거되어도 돌아간다라고 생각한다면 `@Async`가 먼저가 되면 좋을 것 같네요 :)